lib/Basics/FileUtils.cpp

```diff
@@ -421,28 +419,20 @@ bool copyDirectoryRecursive(
   size_t const srcPrefixLength = src.size();
 
 #ifdef TRI_HAVE_WIN32_LIST_FILES
-  struct _wfinddata_t oneItem;
+  _finddata_t oneItem;
   intptr_t handle;
 
-  std::string rcs;
   std::string flt = source + "\\*";
 
-  icu::UnicodeString f(flt.c_str());
```

Comment:
I think this removes our support for utf-8 characters in filenames on windows. This will introduce a regression.

Reply:
no.